utils/tracing/win/Start-Tracing.ps1

```diff
@@ -0,0 +1,28 @@
+param(
+   [bool]$IncludeGeneralTrace = $False

```

Comment:
I'm not a pwsh expert, but can you add some more info to these?
```
[Parameter(ParameterSetName='Default', Position=0, ValueFromPipeline=$true, ValueFromPipelineByPropertyName=$true)]
    [bool]
    ${IncludeGeneralTrace},
```
Not exactly those, but let's add all the validation and help we can to make it easy to use.
That's one of the advantages over bash!

Reply:
Me either :) Did some refactorings .. Let me know your thoughts on the new iteration ..